app_utils/sections/experiment.py

```diff
@@ -33,16 +34,17 @@
     get_problem_types,
     get_ui_elements,
     get_unique_name,
-    make_label,
     parse_ui_elements,
     remove_model_type,
     set_env,
     start_experiment,
 )
 from app_utils.wave_utils import busy_dialog, ui_table_from_df, wave_theme
+from llm_studio.python_configs.text_causal_language_modeling_config import (
+    ConfigProblemBase,
```

Comment:
Can you remove this import? We will probably add new problem types in the future (see also PRs).

Reply:
Ok, but how to type annotate then for the new function I put? Shall I just remove the annotation?

Reply:
For now, we can remove the type annotation. In general, having a `ConfigProblemBase` base class that each problem type subclasses  may be useful.

Reply:
done https://github.com/h2oai/h2o-llmstudio/pull/316/commits/49d422791fe970426297e600acd04e9745d1f440